service.py

```diff
@@ -315,9 +315,9 @@ def get_subject(subjectID):
     ).all()
     for x in a:
         hot_article.append(x)
-
-    return render_template('subject.html', url=url, subject_id=subject.id, articles=articles, hot_article=hot_article, Tool=Tool)
-
+    if not subject.pid  == "None" :
+        return render_template('subject.html', url=url, subject_id=subject.id,lasturl="http://127.0.0.1:5000/subject/"+str(subject.pid) ,articles=articles, hot_article=hot_article, Tool=Tool)
+    else :return render_template('subject.html', url=url, subject_id=subject.id,lasturl="http://127.0.0.1:5000/" ,articles=articles, hot_article=hot_article, Tool=Tool)
```

Comment:
@mmmmmkj Will the above two lines work for a web server whose IP address is not **127.0.0.1**?

-Hui